### DISCUSSION TITLE
Send logs to Database

### DISCUSSION THREAD

**Discussion by jerebear12** [seaweedfs/seaweedfs#3794] [Q&A] [answered]
### Is there a way to send all logs created with glog to a database?
I'm trying to create a dashboard for Seaweed that can save and view the logs.
I have an api that would receive the data and place it in the db. 
 

I'm looking at:
1. modifying/replacing the glog package code used in seaweed
2. replacing glog functions with my own logging functions
3. reading program output and then operating on it

**Comment by kmlebedev** [comment]
Why doesn't filebeat work for you?
https://logit.io/sources/configure/mysql/

**Reply by jerebear12** [reply]
We are not using MySQL. We are using ScyllaDB.

**Comment by jerebear12** [comment, marked as answer]
I am now using a Python script to read and parse the log files by checking to see if the files start with the word 'weed'. Then I'm offering an API route written in Python that when called, will parse the logs and send them to [PocketBase](https://pocketbase.io). I made a small dashboard using Svelte + Sveltekit.
